components/inputs/test/input-date-time-range.visual-diff.html

```diff
@@ -58,7 +58,7 @@
 				<d2l-input-date-time-range id="start-end-label" start-label="A long start date label explanation" end-label="Finish"></d2l-input-date-time-range>
 			</div>
 			<div class="visual-diff">
-				<d2l-input-date-time-range id="start-end-value" start-value="2020-12-02T15:00:00.000Z" end-value="2021-01-12T08:30:00.000Z"></d2l-input-date-time-range>
+				<d2l-input-date-time-range id="start-end-value" start-value="2020-12-02T15:00:00.000Z" end-value="2021-01-12T07:30:00.000Z"></d2l-input-date-time-range>
```

Comment:
I moved this back an hour so that the visual-diffs could capture a change in date due to timezone for the timezone change test